src/preferences/dialog/dlgprefmixerdlg.ui

```diff
@@ -570,6 +581,18 @@
       <string>Main EQ</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
+
+      <item>
+       <widget class="QCheckBox" name="CheckBoxInstantMainEQ">
+        <property name="toolTip">
+         <string>EQ parameters are applied instantly without clicking Apply</string>
+        </property>
+        <property name="text">
+         <string>Apply EQ parameters instantly</string>
```

Comment:
We are in beta-state, and there are only 5 weeks left. This is too short to get this translated to a significant degree. 

Is this feature important enough for this? For me it was OK to have that instantly applied. Who will have benefit from Applying it after "Apply" only? Is there a real use case?